pkg/reconciler/instances/istio/actions/performer_test.go

```diff
@@ -137,6 +137,7 @@ func Test_DefaultIstioPerformer_Install(t *testing.T) {
 		proxy := proxymocks.IstioProxyReset{}
 		provider := clientsetmocks.Provider{}
 		provider.On("GetIstioClient", mock.Anything).Return(ctrlClient, nil)
+		provider.On("RetrieveFrom", mock.AnythingOfType("string"), mock.AnythingOfType("*zap.SugaredLogger")).Return(fake.NewSimpleClientset(), nil)
```

Comment:
Do we need additional tests to verify the CNI logic in performer?

Reply:
you are right, I have forgotten to add tests to performer, should be good now :)